### DISCUSSION TITLE
Pipe flow example

### DISCUSSION THREAD

**Discussion by ahmedelhanafy2022** [UvaCsl/HemoCell#96] [General discussion]
Hello 
1-Regarding the pipe flow example, is it valid for a bifurcated pipe (two branches for example)?
2- If the bifurcated geometry is valid in the pipe flow example, how is the Reynolds number calculated? (or what is the diameter that will be used to calculate the Reynolds number?)
Thank you

**Comment by gzavo** [comment]
Hi,
Please try to define your questions more clearly.
What do you mean by "valid"? If you change the geometry, you also need to change the code to set up the appropriate boundaries.
For the Re, the diameter that you define. Currently for the pipeflow the diameter it is defined in the config.xml, but you can change that in the code as well.
Bests,
Gabor
